player/command.c

```diff
@@ -4786,6 +4786,25 @@ static bool check_property_autorepeat(char *property,  struct MPContext *mpctx)
     return true;
 }
 
+// Whether changes to this property (add/cycle cmds) benefit from cmd->scale
+static bool check_property_scalable(char *property, struct MPContext *mpctx)
+{
+    struct m_option prop = {0};
+    if (mp_property_do(property, M_PROPERTY_GET_TYPE, &prop, mpctx) <= 0)
+        return true;
+
+    // These properties are backed by a floating-point number
+    if (prop.type == &m_option_type_float ||
+        prop.type == &m_option_type_double ||
+        prop.type == &m_option_type_time ||
+        prop.type == &m_option_type_aspect)
```

Comment:
Normally something I'd like to avoid.

Also, this could just be done as a single return statement containing the expression.